crypto/pem/pem_pkey.c

```diff
@@ -156,6 +157,20 @@ int PEM_write_bio_PrivateKey(BIO *bp, EVP_PKEY *x, const EVP_CIPHER *enc,
   return PEM_write_bio_PKCS8PrivateKey(bp, x, enc, (char *)kstr, klen, cb, u);
 }
 
+static int i2d_PrivateKey_void(const void *key, uint8_t **out) {
+  return i2d_PrivateKey((const EVP_PKEY *)key, out);
+}
+
+int PEM_write_bio_PrivateKey_traditional(BIO *bp, EVP_PKEY *x,
+                                         const EVP_CIPHER *enc,
+                                         unsigned char *kstr, int klen,
+                                         pem_password_cb *cb, void *u) {
+  char pem_str[80];
+  BIO_snprintf(pem_str, 80, "%s PRIVATE KEY", x->ameth->pem_str);
```

Comment:
does `x->ameth->pem_str` contain any of the actual key material? we should `OPENSSL_cleanse` `pem_str` before returning if so.

Reply:
No, it just points to a string representing the key type (added in https://github.com/aws/aws-lc/commit/3662fc13a4c4bccadb40a55218d4e71a491f1e5b).
But that does remind me that I should do an assertion on `x->ameth->pem_str` and `x->ameth` here.